keyboards/atlantis/ps17/rules.mk

```diff
@@ -0,0 +1,2 @@
+LTO_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```

Comment:
This can't be enabled at the keyboard level, as it's dependent on keymap specific things. So it should be moved to `keymaps/*/rules.mk`.